sprint-planning/estimation/readme.md

```diff
@@ -1,8 +1,93 @@
-# Estimation
+# Goals for Estimation
 
-## Content TODO's
+When a team adopts agile, as the team works together, the process of estimating
+new work becomes more effective. Over time, they get a sense for how their team
+approaches the stories and the effort that will be required to complete each story.
+With a few successful sprints completed, the team can start to see patterns of
+success and where the team has had failures. The team can start to look at their
+velocity against their estimates and begin to predict with greater accuracy what
+effort will be needed to complete the next story.
 
-Recommend sizing scale such as fibonacci
-Pick stories to establish baseline (example below assumes fibnonacci)
-    - What story represents 5
-    - What story represents 2
+If the makeup of the team is regularly changing, the process of estimation becomes
+challenging as the patterns that drive the estimation process also change. Initially,
+it will take three to five or more sprints to find a good rhythm to the team estimations.
+When the team changes to a new project or a team is reorganized to join with another
+team, the estimation model is essentially reset and a new baseline must be formed.
+
+## Sizing
+
+There are many ways to size stories. This is usually where teams get hung up on
+finding the balance between sizing that feel "willy nilly" and accurately estimating
+the hours to complete the work. It is helpful to remember that this exercise is
+to accurately determine what work can be completed in the sprint...not a precise
+time allocation for the work to be complete.
+
+To determine sizing, the team will generally pick two or three stories that reflect
+a known amount of work of varying size. Usually, a simple, an average and a complex
+story will work best. Each story will be given a relative weight based on whether
+it is more like one of the "example" stories. Over time, the "example" stories may
+need to be revisited if the team feels that all of their stories are average or
+complex.
+
+### Planning Poker
+
+Possibly the most popular approach to sizing is called Planning Poker. This approach
+assigns sizing based on the fibonacci numbers 1,2,3,5,8,13,20,40. Teams will go so
+far as to purchase cards that reflect the numbers and distribute the cards during
+the planning session.
+
+The team would usually pick a simple story and assign it a value of 2. Then the
+team would pick another story that was more difficult and assign it a corresponding
+value...i.e. a 5 would be assigned to a story that is viewed to be more than twice

```

Comment:
That's a team decision, and it will vary team to team. This is why when the team changes significantly, they technically should reset what the numbers mean.